course_info/static/course_info/js/controllers/DetailsController.js

```diff
@@ -0,0 +1,110 @@
+// TODO - this is pretty much a copy of the people controller with some mods.
+// TODO - this still needs a lot of clean up, we don't need everything in here
+
+(function () {
+    var app = angular.module('CourseInfo');
+    app.controller('DetailsController', DetailsController);
+
+    function DetailsController($scope, $routeParams, courseInstances, $compile,
+                               djangoUrl, $http, $q, $log, $uibModal, $sce) {
+
+        dc = this;
+
+        dc.handleAjaxError = function (data, status, headers, config, statusText) {
+            $log.error('Error attempting to ' + config.method + ' ' + config.url +
+                ': ' + status + ' ' + statusText + ': ' + JSON.stringify(data));
+        };
+
+        dc.handleLookupResults = function(results) {
+            var courseResult = results[0];
+            var membersResult = results[1];
+
+            //check if the right data was obtained before storing it
+            if (courseResult.data.course_instance_id == dc.courseInstanceId) {
+                courseInstances.instances[courseResult.data.course_instance_id] = courseResult.data;
+                dc.courseInstance = dc.getFormattedCourseInstance(courseResult.data, membersResult.data)
+            } else {
+                $log.error(' CourseInstance record mismatch for id :'
+                    + dc.courseInstanceId + ',  fetched record for :' + courseResult.data.id);
+            }
+        };
+
+        dc.setCourseInstance = function (id) {
+
+            var course_url = djangoUrl.reverse(
+                'icommons_rest_api_proxy',
+                ['api/course/v2/course_instances/' + id + '/']);
+
+            var members_url = djangoUrl.reverse(
+                'icommons_rest_api_proxy',
+                ['api/course/v2/course_instances/'
+                + id + '/people/']);
+
+            var coursePromise = $http.get(course_url)
+                .error(dc.handleAjaxError);
+
+            var membersPromise = $http.get(members_url)
+                .error(dc.handleAjaxError);
+
+            $q.all([coursePromise, membersPromise])
+                .then(dc.handleLookupResults);
+        };
+
+        dc.stripQuotes = function(str){
+            // soem fields are coming over with quotes around them and those quotes
```

Comment:
Good point, if this is a feed issue it should be fixed there. I'm thinking of pulling this from the code and letting the quotes come through. We can look at fixing either the feed or the data if needed.